### ISSUE TITLE
Pause + options screen

### ISSUE THREAD

**Issue by Sebeb** [dhubber/mazed-up#16]
A basic pause screen which simply allows the player to either return to the menu, change options, or quite the game.

In terms of the options, the only thing I can think of which could be good is an option to show or hide the player's current time.

**Comment by dhubber** [comment]
Actually, I did have an idea about this.  We were talking about the player needing to either remember or write down (on paper) what the pills do.  I'm not a fan of players needing to rely on the second one and most people won't have a good enough memory to do the first (although that is a skill in itself I suppose).  One other option is the Pause screen, as well as various other options (such as toggle timer and I guess toggle sound effects if they find them too creepy :-) ) there could be a (picture) list of all the already collected pills as a small memory aid (maybe they can't directly remember the colours but can remember the order they picked them up).  A second further extension of this would be the option to add a small 'note' in this screen if they wish (i.e. describing what the pill does).  It would be preferential to needing to finding a pen and paper I feel.  Any thought on that?

**Comment by Sebeb** [comment]
Funny, I actually had exactly the same idea in regards to having the pills taken in the pause menu and allowing the player to make notes. It could work quite nicely, stylistically, if every pill the player has taken is displayed, chaotically strewn out across the pause menu, with a simple Rigidbody attached to each pill, so they can be dragged around and played with. And then clicking on a pill could allow the player to type a note directly on the pill, maybe we could even attach a little label to pills the player's notated and have the note written in a handwritten style font? :thinking:

**Comment by dhubber** [comment]
Yes, I was thinking about trying to implement a basic notepad this morning, but the pill copy sprite/colour issue (that I mentioned in the last pill issue) stopped me in my tracks.  As I said there, I think this simple notepad idea is what should principally be in the pause menu.  Pressing pause once is preferable to going through multiple menu layers to find this info.

If you have a suggestion for the pill copy issue, or even a direct solution, I can try this later!  :-)

**Comment by Sebeb** [comment]
Okay, I overhauled the whole pill properties system, which should hopefully have fixed that issue (see last pill issue.)
Definitely, agree that this should be accessible immediately upon the game being paused. I was thinking about how we could make this controller friendly actually, I was wondering if it may be possible to assign a symbol to each pill. So we'd have 19 symbols which loosely represent each side effect, and selecting a pill would allow the player to either click on a symbol which they think matches the pill's side-effect, or select it from some kind of radial menu on a controller. This could also allow us to easily display any assigned symbols on pills in game, so the player doesn't have to quickly pause and compare sprites since the symbols would all be relatively small but hopefully recognizable.
